proj/src/main/webapp/chat.js

```diff
@@ -830,9 +863,13 @@ function addUserInfoToDom(userObj) {
     }
 
     profile.querySelector("#user-display-name").innerText = userObj.fname + ' ' + userObj.lname;
-    if (userObj.photo != null) {
-        profile.querySelector("#user-pfp").src = userObj.photo;
-    }
+
+    const url = userObj.photo || "gs://arringtonh-step-2020-d.appspot.com/default.png";
```

Comment:
can the default image be put in a constant and referenced from that constant so that it only needs to change in one  place when it changes.